ros2param/test/test_verb_dump.py

```diff
@@ -211,6 +210,8 @@ def test_verb_dump_invalid_path(self):
             strict=True
         )
 
+    # Flaky on Galactic: https://github.com/ros2/ros2cli/issues/630
+    @pytest.mark.xfail
```

Comment:
Can we do the following?

```suggestion
    @pytest.mark.xfail(reason="Flaky on Galactic: https://github.com/ros2/ros2cli/issues/630")
```